### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by Ag0n0s** [Mezryss/FVTT-Genesys#46] [Ideas]
Hello,
In the future will we be able to translate into our language on the character sheet  words "soak value", "wounds", "strain", "defense", "brawn", "agility", "intellect", "cunning "," Willpower "," presence "?
Thank you

**Comment by Mezryss** [comment]
People will need to write translations the same way they do for any Foundry system. As it is right now it's already translatable (though I think I may have hard-coded some strings, which I'll need to fix at some point), though a bit disorganized.

**Comment by Cristalnoir** [comment]
I am transmitting a small translation error in FR. Column 16 009, "Qualitées" must be changed to "Attributs".
